### DISCUSSION TITLE
Cooldown Sweep Animation not Showing

### DISCUSSION THREAD

**Discussion by Kezwo** [SFX-WoW/Masque#190] [Issues] [answered]
Since version 8.1.1-alpha, the cooldown sweep animations don't work anymore for icons in TellMeWhen. This affects both skill cooldowns and the global cooldown. In other addons, like Dominos and WeakAura, the animation still works fine. Not sure if this is an issue with Masque or maybe TMW needs to adapt to the new version, but it was still working with TMW in Masque version 8.1.0-alpha, and I did not change anything and TMW did not recieve any update in the last few days. Also, when I disable Masque, TMW shows the game's sweep animations as usual.

**Comment by ZinojoDev** [comment]
This appears to be a framelevel issue. The cooldown sweep is still displaying, but is covered by the icon.

The following code in 

> TellMeWhen\Components\IconViews\Icon\Icon.lua line 153

```lua
	local frameLevelOffset = isDefaultSkin and 1 or -2
	
	if CooldownSweep then
		CooldownSweep.cooldown:SetFrameLevel( icon:GetFrameLevel() + 0 + frameLevelOffset)
	end
```

is causing the issue. Changing the -2 to 1 should fix it for your local copy.

**Comment by StormFX** [comment]
I added a line in the change log noting that Masque no longer adjusts the frame levels of buttons or child frames. Not sure if that's relevant, but nonetheless.

**Comment by Kezwo** [comment, marked as answer]
This worked, thank you. I will open a ticket on the TMW issue tracker and let the author know.

**Comment by StormFX** [comment]
Closing.
